app/presenters/register_api/single_application_presenter.rb

```diff
@@ -27,6 +27,9 @@ def as_json
           course: course_info_for(application_choice.current_course_option),
           qualifications:,
           hesa_itt_data: hesa_itt_data.presence,
```

Comment:
I can't quite put this comment on the right line, but line 24 -- we need to allow nil for recruited_at.

Reply:
Good shout - we probably want to add the `accepted_at` here too, this is the timestamp when the Offer was accepted